packages/cache/src/models/Subscribable.ts

```diff
@@ -0,0 +1,35 @@
+import type { CacheKey, CacheOptions, Sync } from '../types'
+import { hashCacheKey } from '../utils'
+
+export class Subscribable {
```

Comment:
I chose to name the class `Subscribable` because it provides subscription functionality and manages subscribers, including synchronization. I believe the focus should be on managing subscribers rather than on synchronization itself, as synchronization is a part of subscriber management. Therefore, I decided on the name `Subscribable`, which better reflects this aspect from an external perspective.